scripts/test.sh

```diff
@@ -5,7 +5,7 @@
 #   - go vet        (https://golang.org/cmd/vet)
 #   - test coverage (https://blog.golang.org/cover)
 
-COVERALLS_TOKEN=t47LG6BQsfLwb9WxB56hXUezvwpED6D11
+COVERALLS_REPO_TOKEN=t47LG6BQsfLwb9WxB56hXUezvwpED6D11
```

Comment:
OK so I removed COVERALLS_TOKEN from test.sh and I added additional condition to post test post test coverage to Coveralls. 
Earlier tests for pull requests always fail because COVERALLS_REPO_TOKEN is unset for pull requests...